src/applications/vaos/tests/appointment-list/components/PastAppointmentsListV2.unit.spec.jsx

```diff
@@ -0,0 +1,321 @@
+      new RegExp(pastDate.tz('America/Denver').format('dddd, MMMM D'), 'i'),
+    );
+
+    const firstCard = screen.getAllByRole('listitem')[0];
+
+    const timeHeader = within(firstCard).getAllByText(
+      new RegExp(pastDate.tz('America/Denver').format('h:mm'), 'i'),
+    )[0];
+
+    expect(screen.queryByText(/You don’t have any appointments/i)).not.to.exist;
+    expect(firstCard).not.to.contain.text('Canceled');
+    expect(firstCard).to.contain.text('VA appointment');
+
+    expect(timeHeader).to.contain.text('MT');
+    expect(timeHeader).to.contain.text('Mountain time');
+  });
+
+  it('should show information with facility name', async () => {
+    const appointment = getVAAppointmentMock();
+    appointment.attributes = {
+      ...appointment.attributes,
+      startDate: pastDate.format(),
+      clinicFriendlyName: 'C&P BEV AUDIO FTC1',
+      facilityId: '983',
+      sta6aid: '983GC',
+    };
+    appointment.attributes.vdsAppointments[0].currentStatus = 'CHECKED OUT';
+    mockPastAppointmentInfo({ va: [appointment] });
+
+    const facility = {
+      id: 'vha_442GC',
+      attributes: {
+        ...getVAFacilityMock().attributes,
+        uniqueId: '442GC',
+        name: 'Cheyenne VA Medical Center',
+        address: {
+          physical: {
+            zip: '82001-5356',
+            city: 'Cheyenne',
+            state: 'WY',
+            address1: '2360 East Pershing Boulevard',
+          },
+        },
+        phone: {
+          main: '307-778-7550',
+        },
+      },
+    };
+    mockFacilitiesFetch('vha_442GC', [facility]);
+
+    const screen = renderWithStoreAndRouter(<PastAppointmentsListV2 />, {
+      initialState,
+      reducers,
+    });
+
+    await screen.findAllByText(
+      new RegExp(pastDate.tz('America/Denver').format('dddd, MMMM D'), 'i'),
+    );
+
+    const firstCard = screen.getAllByRole('listitem')[0];
+
+    expect(
+      within(firstCard).getByText(
+        new RegExp(pastDate.tz('America/Denver').format('h:mm'), 'i'),
+      ),
+    ).to.exist;
+    expect(within(firstCard).getByText(/Cheyenne VA Medical Center/i)).to.exist;
+    expect(screen.baseElement).not.to.contain.text('VA appointment');
+  });
+
+  // Skipped until cancelled functionality added
+  it.skip('should have correct status when previously cancelled', async () => {
+    const appointment = getVAAppointmentMock();
+    appointment.attributes = {
+      ...appointment.attributes,
+      startDate: pastDate.format(),
+      clinicFriendlyName: 'Some clinic',
+      facilityId: '983',
+      sta6aid: '983GC',
+    };
+    appointment.attributes.vdsAppointments[0].currentStatus =
+      'CANCELLED BY PATIENT';
+    mockPastAppointmentInfo({ va: [appointment] });
+
+    const screen = renderWithStoreAndRouter(<PastAppointmentsListV2 />, {
+      initialState,
+      reducers,
+    });
+
+    await screen.findByText(
+      new RegExp(pastDate.tz('America/Denver').format('MMMM YYYY'), 'i'),
+    );
+
+    screen.debug();
```

Comment:
Probably can remove this

Reply:
Yes, suprised the linter didn't catch that. Removed.